### PR TITLE
DOC-827 Add support for parquet format to BigQuery output

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves [DOC-827](https://redpandadata.atlassian.net/browse/DOC-827)
Review deadline: 11th December

## Page previews

[gcp_bigquery output](https://deploy-preview-124--redpanda-connect.netlify.app/redpanda-connect/components/outputs/gcp_bigquery/#format)

## Checks

- [ ] New feature
- [x] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

[DOC-827]: https://redpandadata.atlassian.net/browse/DOC-827?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/components/pages/outputs/gcp_bigquery.adoc [hunk 1]
```diff
@@ -122,6 +125,10 @@ The same is true for the CSV format.
 
 When the field `csv.header` is specified for the `CSV` format, a header row is inserted as the first line of each message batch. If this field is not provided, then the first message of each message batch must include a header line.
 
+=== Parquet
+
+You can use the xref:components:processors/parquet_encode.adoc[`parquet_encode` processor] to convert message data into a Parquet format before it is sent to this output. Each message sent by the processor must be a full Parquet message.
```
Comment (asimms41) [inline modules/components/pages/outputs/gcp_bigquery.adoc:130]: @rockwotj What do you mean by a full message and how do you guarantee they are created.

Comment (rockwotj) [reply modules/components/pages/outputs/gcp_bigquery.adoc:130]: Here is an example. The output must be sent parquet files, which you can create from a batch using `parquet_encode`
```
input:
  generate:
    mapping: |
      root = {
        "foo": random_int(),
        "bar": uuid_v4(),
        "time": now(),
      }
    interval: 0
    count: 1000
    batch_size: 1000
pipeline:
  processors:
    - parquet_encode:
        schema:
          - name: foo
            type: INT64
          - name: bar
            type: UTF8
          - name: time
            type: UTF8
        default_compression: zstd
output:
  gcp_bigquery:
    project: "${PROJECT}"
    dataset: "my_bq_dataset"
    table: "redpanda_connect_ingest"
    format: PARQUET
```

Comment (rockwotj) [reply modules/components/pages/outputs/gcp_bigquery.adoc:130]: full message is probably misleading and you can say something like: "each message to the output must be a parquet file"

Comment (asimms41) [reply modules/components/pages/outputs/gcp_bigquery.adoc]: Thanks for the example. i've updated the text and added the example.

**Review by Feediver1 — APPROVED** [review]
lgtm

**Pull request merged**